modules/Microsoft.Web/hostingEnvironments/configurations-networking/deploy.bicep

```diff
@@ -0,0 +1,53 @@
+@description('Conditional. The name of the parent Hosting Environment. Required if the template is used in a standalone deployment.')
+param hostingEnvironmentName string
+
+@description('Optional. Property to enable and disable new private endpoint connection creation on ASE.')
+param allowNewPrivateEndpointConnections bool = false
+
+@description('Optional. Property to enable and disable FTP on ASEV3.')
+param ftpEnabled bool = false
+
+@description('Optional. Customer provided Inbound IP Address. Only able to be set on Ase create.')
+param inboundIpAddressOverride string = ''
+
+@description('Optional. Property to enable and disable Remote Debug on ASEv3.')
+param remoteDebugEnabled bool = false
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (enableDefaultTelemetry) {
+  name: 'pid-47ed15a6-730a-4827-bcb4-0fd963ffbd82-${uniqueString(deployment().name)}'
+  properties: {
+    mode: 'Incremental'
+    template: {
+      '$schema': 'https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#'
+      contentVersion: '1.0.0.0'
+      resources: []
+    }
+  }
+}
+
+resource appServiceEnvironment 'Microsoft.Web/hostingEnvironments@2022-03-01' existing = {
+  name: hostingEnvironmentName
+}
+
+resource configuration 'Microsoft.Web/hostingEnvironments/configurations@2022-03-01' = {
+  name: 'networking'
```

Comment:
Like in the other comment, this probably HAS to be this value - yet I'd appreciate if we could move this into a parameter with the default value of `networking` to align with other implementations like Storage Account blob services:
```bicep
@description('Optional. The name of the blob service.')
param name string = 'default'
```

Reply:
@AlexanderSehr , I had that initially but got into warning issues about the property being defined as name and also validations are lost for the resource properties. Here's what would happen:

```pwsh
The discriminator property "name" value cannot be determined at compilation time. Type checking for this object is disabled. Bicep(BCP225)
```

```bicep
resource configuration 'Microsoft.Web/hostingEnvironments/configurations@2022-03-01' = {
  name: name // this throws a warning
  parent: appServiceEnvironment
  properties: {
    certificateUrl: certificateUrl
    keyVaultReferenceIdentity: keyVaultReferenceIdentity
    dnsSuffix: dnsSuffix
  }
}
```

Reply:
I get your point. It is however a common issue in the implementation. We get the same warning for example in the Static Sites Config module for the name property - where it is actually variable (either 'appsettings' or 'functionappsettings'). However, the same warning does not occur for the blobervices module.

To this end, I'd still recommend to rather disable the linter rule and push for consistency using `#disable-next-line BCP225 // Disables incorrect error that `name` cannot be determined at compile time.`

We can also query for some additional thoughts by e.g. @eriqua or @MariusStorhaug

Reply:
Ref #2807

Reply:
@AlexanderSehr can we resolve this comment given we're sticking to the current implementation as per the vote?

Reply:
Yes, sorry :)